### DISCUSSION TITLE
Assistance Required with Calibration Interruption Using GRDC Discharge Data

### DISCUSSION THREAD

**Discussion by weixiaoxun** [iiasa/CWatM#163] [Q&A] [answered]
Dear Professor @dof1985 @mikhailsmilovic 

I hope this message finds you well.

I have recently re-downloaded the code from GitHub and used the GRDC discharge data as the observed dataset for the calibration. However, the calibration process was interrupted midway, and I am unsure of the cause.

Could you kindly assist me in identifying what might have gone wrong and provide guidance on how to resolve the issue?

Thank you very much for your time and help.
![微信截图_20241017105543](https://github.com/user-attachments/assets/1477c1bf-03ef-481b-b61c-39a3c22024c3)
![微信截图_20241017110731](https://github.com/user-attachments/assets/26fdb4d2-ec2b-4df0-8b32-1655d884474e)

Best regards,
Weixiaoxun

**Comment by dof1985** [comment]
Hi @weixiaoxun 

Please check the most recent log_files and try to identify the problem - probably the model had crashed for some reason.

**Reply by weixiaoxun** [reply, marked as answer]
I apologize for the oversight. It seems I forgot to update the path for the observed data in the settings, which is likely the reason for the persistent errors I’ve been encountering. I'll correct the path and run the calibration again.

Thank you for your patience, and I appreciate your guidance.

**Reply by dof1985** [reply]
Updating the settings with the correct path is a lesson that everyone can benefit from

**Reply by weixiaoxun** [reply]
Dear @dof1985 ，Unfortunately, the same error has occurred again

**Reply by weixiaoxun** [reply]
[log00_001.txt](https://github.com/user-attachments/files/17412685/log00_001.txt)

[settings_calibration.txt](https://github.com/user-attachments/files/17411652/settings_calibration.txt)

**Comment by dof1985** [comment]
Hi @weixiaoxun 

Sorry - I have over look - the error is actually quite clear. While reading your simulated data - the code were expecting discharge for on station - but instead there are 9 there (or total of 10 columns). 
Please setup the calibration scheme again, but in your settings file you should make sure that you have only one station (x, y) in your gauges setting. 

If you need to calibrate multiple station you should currently run them as separate processes.

**Reply by weixiaoxun** [reply]
Thank you very much for your response. I now understand the issue. I have been using the LDD tif file for my observation stations. From your explanation, it seems that the observation stations should be represented using coordinates, is that correct?
